.gitmodules

```diff
@@ -0,0 +1,33 @@
+[submodule "crt/aws-c-common"]
+	path = crt/aws-c-common
+	url = https://github.com/awslabs/aws-c-common
+[submodule "crt/aws-c-auth"]
+	path = crt/aws-c-auth
+	url = https://github.com/awslabs/aws-c-auth
+[submodule "crt/aws-c-cal"]
+	path = crt/aws-c-cal
+	url = https://github.com/awslabs/aws-c-cal
+[submodule "crt/aws-c-compression"]
+	path = crt/aws-c-compression
+	url = https://github.com/awslabs/aws-c-compression
+[submodule "crt/aws-c-http"]
+	path = crt/aws-c-http
+	url = https://github.com/awslabs/aws-c-http
+[submodule "crt/aws-c-io"]
+	path = crt/aws-c-io
+	url = https://github.com/awslabs/aws-c-io
+[submodule "crt/aws-checksums"]
+	path = crt/aws-checksums
+	url = https://github.com/awslabs/aws-checksums
+[submodule "crt/aws-c-mqtt"]
+	path = crt/aws-c-mqtt
+	url = https://github.com/awslabs/aws-c-mqtt
+[submodule "crt/s2n"]
+	path = crt/s2n
+	url = https://github.com/aws/s2n-tls
+[submodule "crt/aws-lc"]
+	path = crt/aws-lc
+	url = https://github.com/aws/aws-lc
+[submodule "crt/aws-c-sdkutils"]
+	path = crt/aws-c-sdkutils
+	url = https://github.com/awslabs/aws-c-sdkutils
```

Comment:
**Question**: Does the addition of submodules here mean they need to be kept in sync with the versions referenced in **aws-crt-java**?

Reply:
I don't think they _have_ to be kept in sync necessarily since their usage is independent. We probably don't want them to drift too much but I don't think I'm overly concerned at the moment if they differ.